pages/common/pkgz.md

```diff
@@ -0,0 +1,24 @@
+# pkgz
+
+> Unified package manager CLI for installing, removing, updating, and searching packages from multiple sources.  
+> More information: <https://github.com/roguehashrate/pkgz>
```

Comment:
```suggestion
> More information: <https://github.com/roguehashrate/pkgz>
```
```suggestion
> More information: <https://github.com/roguehashrate/pkgz#-usage>
```
This will make it easier for users to look up usage instructions when the link takes them directly to it